### PR TITLE
Add getzim.sh script

### PR DESCRIPTION
**Pull request by mkg20001**
The script allows the user to pick a source and download it from cli without having to visit http://wiki.kiwix.org/wiki/Content_in_all_languages
It also automatically resumes the download and verifies the md5sum

### CODE REVIEW

**Comment by momack2** [comment]
Seems related to https://github.com/ipfs/distributed-wikipedia-mirror/issues/58?

File: getzim.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+if [ ! -e ".content" ]; then
+  echo "Downloading content list..."
+  curl -s http://wiki.kiwix.org/wiki/Content_in_all_languages > .content
```
Comment (lidel) [inline getzim.sh]: It did not work for me unless I enabled following redirect to HTTPS:
```suggestion
  curl -sL http://wiki.kiwix.org/wiki/Content_in_all_languages > .content
```

Comment (kelson42) [reply getzim.sh]: Maybe just better use the https version https://wiki.kiwix.org/wiki/Content_in_all_languages

**Comment by mkg20001** [comment]
I've changed the script to work with the download.kiwix.org site directly

There's a command `bash getzim.sh choose` that allows to create a download command

One of these for example is `bash getzim.sh download other archlinux en all notag any` which downloads the english archlinux zim

Note that currently SHA256 validation is broken and needs to be fixed

**Comment by mkg20001** [comment]
The url command did already exist (`download_url`), but now I've aliased it to url as well and documented it

Any is now latest

**Comment by mkg20001** [comment]
Now the url command also returns json and prints the log to stderr

File: getzim.sh [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+fetch_with_cache() {
+  OUTFILE=${1/"/"//"_"}
+  OUTFILE="$CACHE/_page$OUTFILE"
+
+  if [ -e "$OUTFILE" ]; then
+    cat "$OUTFILE"
+  else
+    OUT=$(curl -sL "$BASEURL$1")
+    echo "$OUT" > "$OUTFILE"
```
Comment (lidel) [inline getzim.sh]: When executing `url` without doing cache update first I got error:

```console
getzim.sh: line 49: /home/lidel/project/ipfs/distributed-wikipedia-mirror/.cache/_pagewikipedia: No such file or directory
```

Fix:

```suggestion
    mkdir -p "$CACHE"
    echo "$OUT" > "$OUTFILE"
```

**Comment by mkg20001** [comment]
The cache being filled without cache_update was sorta a unintended feature, but I think we can just make this a proper feature

**Comment by lidel** [comment]
@mkg20001 anything you want to add, or ok to merge? (can be a separate PR)

**Comment by mkg20001** [comment]
Not really. Don't fix it, if it ain't broken :P

**Comment by lidel** [comment]
Thanks again!

**Pull request merged**